animation_nodes/nodes/falloff/sound_falloff.pyx

```diff
@@ -0,0 +1,194 @@
+import bpy
+import numpy
+from bpy.props import *
+from ... base_types import AnimationNode
+from ... algorithms.interpolations import Linear
+from ... data_structures cimport DoubleList, FloatList, BaseFalloff
+from . interpolate_list_falloff import createIndexBasedFalloff, createFalloffBasedFalloff
+
+typeItems = [
+    ("AVERAGE", "Average", "", "FORCE_TURBULENCE", 0),
+    ("SPECTRUM", "Spectrum", "", "RNDCURVE", 1)
+]
+
+spectrumFalloffTypeItems = [
+    ("INDEX_FREQUENCY", "Index Frequency", "NONE", 0),
+    ("FALLOFF_FREQUENCY", "Falloff Frequency", "NONE", 1)
+]
+
+indexFrequencyExtensionItems = [
+    ("LOOP", "Loop", "", "NONE", 0),
+    ("MIRROR", "Mirror", "", "NONE", 1),
+    ("EXTEND", "Extend Last", "NONE", 2)
+]
+
+reductionFunctionItems = [
```

Comment:
some of these options are shared with other sound nodes, aren't they? Might be better to import them from there.